# Author: julian.morillo@bsc.es
# Make the test more tolerant (needed for RISC-V, as this target is not expected to emit optimization remarks)
--- flang/test/Driver/optimization-remark.f90.ORIG	2025-07-15 17:05:36.533880333 +0200
+++ flang/test/Driver/optimization-remark.f90	2025-07-15 17:11:17.597882229 +0200
@@ -19,7 +19,7 @@
 ! RUN: %flang %s -O1 -Rpass=loop -S %{output} 2>&1 | FileCheck %s --check-prefix=PASS-REGEX-LOOP-ONLY
 
 ! Check valid -Rpass-missed regex
-! RUN: %flang %s -O1 -Rpass-missed=loop -S %{output} 2>&1 | FileCheck %s --check-prefix=MISSED-REGEX-LOOP-ONLY
+! RUN: %flang %s -O1 -Rpass-missed=loop -S %{output} 2>&1 | FileCheck %s --allow-empty --check-prefix=MISSED-REGEX-LOOP-ONLY
 
 ! Check valid -Rpass-analysis regex
 ! RUN: %flang %s -O1 -Rpass-analysis=loop -S %{output} 2>&1 | FileCheck %s --check-prefix=ANALYSIS-REGEX-LOOP-ONLY
